### DISCUSSION TITLE
Using methods in rewrites

### DISCUSSION THREAD

**Discussion by PierBover** [vercel/vercel#3775] [Ideas]
The docs recommend using rewrites instead of routes but AFAIK it's not possible to use methods in the rewrite config.

I tried it with `now dev` and it stoped with this error:

> Error! Invalid `rewrites` property: [0] should NOT have additional properties

I'm guessing this feature has not been implemented and if that's the case please consider this a feature request.

**Comment by mcsdevv** [comment]
Hey @PierBover, an update on this. `methods` will be added to the `rewrites` option soon.
